Week3/homework/step3.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 function createBase(base) {
-  // Replace this comment and the next line with your code
-  console.log(base);
+  return addition => base + addition;
```

Comment:
Nice use of an arrow function. As we saw in class, sometimes you this as:

```js
const createBase = base => addition => base + addition;
```

That takes some getting used to.
```